src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -70,7 +72,7 @@
     private static final Logger logger = Logger.getLogger(GhprbTrigger.class.getName());
 
     private static final ExecutorService pool = Executors.newFixedThreadPool(
```

Comment:
Yeah, but I don't run a 32 core master server and tend to use large machines as agents.  If a large thread pool is a concern then I don't mind keeping the same values as before and just updating how the property is actually read.